api/r/cellxgene.census/tests/testthat/acceptance-tests.R

```diff
@@ -102,7 +102,7 @@ test_that("test_incremental_read_X_human", {
 })
 
 test_that("test_incremental_read_X_human-large-buffer-size", {
-  census <- open_soma_latest_for_test(soma.init_buffer_bytes = paste(4 * 1024**3))
+  census <- open_soma_latest_for_test(soma.init_buffer_bytes = paste(1 * 1024**3))
```

Comment:
on very high cpu count machines, this will OOM due to per-core buffer allocation.  Reducing to a GiB resolves the issue